setup.py

```diff
@@ -46,8 +49,6 @@
 ]
 if PY_VERSION_10_OR_LATER:
     test_requirements.append("importlib-metadata>=5.2.0")
-if PY_VERSION_11_OR_LATER:
-    test_requirements.append("typed-ast>=1.5.4")
```

Comment:
This is no longer need in Python 3 and it cannot compile with Python 3 AFAICT. I could not find this library being used.